src/core/data.js

```diff
@@ -272,22 +272,5 @@ module.exports = function (context) {
     }
   };
 
-  data.save = function (cb) {
-    const type = context.data.get('type');
-    if (type === 'github') {
-      source.github.save(context, cb);
-    } else if (type === 'gist') {
-      source.gist.save(context, cb);
-    } else if (type === 'local') {
```

Comment:
Interesting - did `local` ever work? Does that refer to in-memory or to a downloaded file?

Reply:
No idea, all of the file downloads have their own code path, so not sure what `type` local ever was.